.github/workflows/scheduled-link-checker.yml

```diff
@@ -0,0 +1,14 @@
+on:
```

Comment:
Is it possible, if not set yet, to sends us a message on slack whenever this fails?

Reply:
I'm not sure if this is possible out of the box. But there looks to be some GHA for slack notifications that could help. I say we hold on slack setup for now. 

We will for sure get a GitHub failure email, and I'm hoping that maybe our github Slack integration in #feed-packer-github handles some of the ci failures :crossed_fingers: